### ISSUE TITLE
improve first steps when installing deps via github 

### ISSUE THREAD

**Issue by timaschew** [jspm/jspm-cli#836]
I had the same issue, when I started with jspm.
In the gitter chat I got help, after two hours there was another guy in the chat with the same question, and on the same day there was a github issue with the same problem: https://github.com/jspm/jspm-cli/issues/835

IMO this is really annoying for jspm beginners.

So maybe add a hint on this page: https://github.com/jspm/jspm-cli/blob/master/docs/installing-packages.md

how to get dependencies working when installing via github: either the package.json should set the `registry` property or use an override.

**Comment by timaschew** [comment]
Maybe even show a note in the terminal, for instance:

```
$ jspm install github:component/dom
      Looking up github:component/dom
      Updating registry cache...

warn  dependencies of github:component/dom will be ignored, reason registry is github

ok    Up to date - component/dom as github:component/dom@^1.0.7 (1.0.7)
```

**Comment by guybedford** [comment]
Good suggestion - I've added this in https://github.com/jspm/github/commit/848aae8f7714bb36d05956268e11b01552f5e92d.

**Comment by guybedford** [comment]
Released in jspm@0.16.0-beta.6.
